src/Sentry.AspNet/Internal/SystemWebVersionLocator.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Reflection;
+using System.Web;
+using Sentry.Internal;
+
+namespace Sentry.AspNet.Internal
+{
+    internal static class SystemWebVersionLocator
+    {
+        internal static string? GetCurrent()
+        {
+            if (ReleaseLocator.GetCurrent() is string release)
+            {
+                return release;
+            }
+            else if (HttpContext.Current?.ApplicationInstance?.GetType() is Type type)
+            {
+                while (type != null && type.Namespace == "ASP")
+                {
+                    type = type.BaseType;
+                }
```

Comment:
This probably needs a comment

Reply:
Added a comment, not a fan of this solution but it seems to be the only way of retrieving the version. (It'll try to get the current version using the original version while this is a fallback for ASP.NET